### PR TITLE
WIP -  Pending dependency upgrade.  APIS-4549 - Use new combined-api-definition

### PR DESCRIPTION
**Pull request by AndySpaven**

### CODE REVIEW

File: app/uk/gov/hmrc/ramltools/Implicits.scala [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/*
```
Comment (adampridmore) [inline app/uk/gov/hmrc/ramltools/Implicits.scala]: This needs deleting!

File: conf/application.conf [hunk 2]
```diff
@@ -80,7 +80,7 @@ platform {
...
+play.filters.headers.contentSecurityPolicy = "default-src 'unsafe-inline' 'self' localhost:9032 data: https://www.google-analytics.com https://www.googletagmanager.com; script-src 'unsafe-inline' 'self' localhost:9032 https://www.google-analytics.com data: https://www.googletagmanager.com https://tagmanager.google.com; font-src 'self' data: https://ssl.gstatic.com https://www.gstatic.com https://fonts.gstatic.com https://fonts.googleapis.com; img-src 'self' https://ssl.gstatic.com https://www.gstatic.com https://www.google-analytics.com https://www.googletagmanager.com data:; style-src 'self' 'unsafe-inline' https://tagmanager.google.com https://fonts.googleapis.com"
```
Comment (adampridmore) [inline conf/application.conf]: I *think* this should be the CSP on the left! I think for some reason this is removing two additional localhost rules. Not sure why it was changed in the branch (did we merge from master or something?)

File: run_local_with_dependencies.sh [hunk 3]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-sm --start API_DEFINITION API_EXAMPLE_MICROSERVICE -f
+sm --start API_DOCUMENTATION SERVICE_LOCATOR API_DEFINITION API_EXAMPLE_MICROSERVICE -f
```
Comment (adampridmore) [inline run_local_with_dependencies.sh]: Should this drop:
- SERVICE_LOCATOR
 - API_DOCUMENTATION

And have
 -  COMBINED_API_DEFINITION

File: conf/application.conf (hunk 2 above)
Comment (AndySpaven) [reply conf/application.conf]: There was a merge from master.  Did some of the CSP corrections get done in this branch ?

File: run_local_with_dependencies.sh (hunk 3 above)
Comment (AndySpaven) [reply run_local_with_dependencies.sh]: That's that flipping revert.

Comment (AndySpaven) [reply run_local_with_dependencies.sh]: Sorted

File: conf/application.conf (hunk 2 above)
Comment (AndySpaven) [reply conf/application.conf]: Sorted. I believe

**Pull request merged**